paasta_tools/cli/cmds/status.py

```diff
@@ -836,232 +801,87 @@ def _print_flink_status_from_job_manager(
-        )
-        output.append(
-            f"      Container Metrics: https://grafana.yelpcorp.com/d/flink-container-metrics/flink-container-metrics?orgId=1&var-datasource=Prometheus-flink&var-region=uswest2-{ecosystem}&var-service={service}&var-instance={instance}&from=now-24h&to=now"
-        )
-        output.append(
-            f"      JVM Metrics: https://grafana.yelpcorp.com/d/flink-jvm-metrics/flink-jvm-metrics?orgId=1&var-datasource=Prometheus-flink&var-region=uswest2-{ecosystem}&var-service={service}&var-instance={instance}&from=now-24h&to=now"
-        )
-
-        # Print Flink Costs Link
-        output.append(
-            f"      Flink Cost: https://app.cloudzero.com/explorer?activeCostType=invoiced_amortized_cost&partitions=costcontext%3AResource%20Summary&dateRange=Last%2030%20Days&costcontext%3AKube%20Paasta%20Cluster={cluster}&costcontext%3APaasta%20Instance={instance}&costcontext%3APaasta%20Service={service}&showRightFlyout=filters"
-        )
-
-        output.append(f"{OUTPUT_HORIZONTAL_RULE}")
-
-    # Print Flink Cluster State
-    color = PaastaColors.green if status["state"] == "running" else PaastaColors.yellow
-    output.append(f"    State: {color(status['state'].title())}")
-
-    # Print Flink Cluster Pod Info
-    pod_running_count = pod_evicted_count = pod_other_count = 0
-    # default for evicted in case where pod status is not available
-    evicted = f"{pod_evicted_count}"
-    for pod in status["pod_status"]:
-        if pod["phase"] == "Running":
-            pod_running_count += 1
-        elif pod["phase"] == "Failed" and pod["reason"] == "Evicted":
-            pod_evicted_count += 1
-        else:
-            pod_other_count += 1
-        evicted = (
-            PaastaColors.red(f"{pod_evicted_count}")
-            if pod_evicted_count > 0
-            else f"{pod_evicted_count}"
-        )
-
-    pods_total_count = pod_running_count + pod_evicted_count + pod_other_count
-    output.append(
-        "    Pods:"
-        f" {pod_running_count} running,"
-        f" {evicted} evicted,"
-        f" {pod_other_count} other,"
-        f" {pods_total_count} total"
+    # Collect instance details
+    instance_details = flink_tools.get_flink_instance_details(
+        metadata, flink_config, flink_instance_config, service
     )
 
-    if not should_job_info_be_shown(status["state"]):
-        # In case where the jobmanager of cluster is in crashloopbackoff
-        # The pods for the cluster will be available and we need to show the pods.
-        # So that paasta status -v and kubectl get pods show the same consistent result.
-        if verbose and len(status["pod_status"]) > 0:
-            append_pod_status(status["pod_status"], output)
-        output.append("    No other information available in non-running state")
-        return 0
-
+    # Get overview and jobs if running
+    overview = None
+    jobs: List[FlinkJobDetails] = []
```

Comment:
i can't comment on L812, so i'll comment here instead: could we merge this branch with the `if status["state"] == "running":` branch above? doesn't look like this depends on `instance_details` and it looks a little funky to have something like
```Python
if status["state"] == "running":
    ...

instance_details = flink_tools.get_flink_instance_details(
    metadata, flink_config, flink_instance_config, service
)

if status["state"] == "running":
    ...
```

instead of 
```Python
if status["state"] == "running":
    ...
    ...

instance_details = flink_tools.get_flink_instance_details(
    metadata, flink_config, flink_instance_config, service
)
```